rinja_derive/src/generator/expr.rs

```diff
@@ -584,6 +586,7 @@ impl<'a> Generator<'a, '_> {
                 return Ok(DisplayWrap::Unwrapped);
             }
         }
+        #[allow(clippy::literal_string_with_formatting_args)]
```

Comment:
Oh that's a new lint I added haha.

Reply:
Thank you, I like the lint! I forgot more than once to surround the string with `format!(…)`. :)